src/zatca/signing/index.ts

```diff
@@ -119,7 +122,9 @@ export const getCertificateInfo = (certificate_string: string): {hash: string, i
  */
 export const cleanUpCertificateString = (certificate_string: string): string => {
     const r = process.platform === "win32" ? "\r" : "";
-    return certificate_string.replace(`-----BEGIN CERTIFICATE-----${r}\n`, "").replace("-----END CERTIFICATE-----", "").trim()
+    
```

Comment:
why did you remove this? it's a fix for windows OpenSSL. 
remove this file from the PR

Reply:
@shadiallouh  check this issue for more details 
https://github.com/wes4m/zatca-xml-js/issues/9